Makefile

```diff
@@ -43,14 +43,15 @@ fmt: ## format the code
 	goimports -local $(PWD)/tests/integration-test/ -w .
 
 dev_docker: ## build docker images for development/testing usages
-	docker build -t scroll_l1geth ./common/docker/l1geth/
-	docker build -t scroll_l2geth ./common/docker/l2geth/
+	docker pull postgres
+	docker build -t scroll_l1geth ./common/docker/l1geth/ --platform linux/amd64
+	docker build -t scroll_l2geth ./common/docker/l2geth/ --platform linux/amd64
```

Comment:
I also encounter this issue, but not sure whether need add this. if some run this on x86_64 maybe have issue

Reply:
Got it. I change this because CI and Mac can both use these commands. forget to consider a broader use case.

Reply:
but I think we only added images of `linux/amd64`? thus can keep this. cc @Thegaram for more context about image building.

Reply:
removed.